Lib/test/test_memoryio.py

```diff
@@ -477,6 +479,29 @@ def test_getbuffer_empty(self):
         buf2.release()
         memio.write(b'x')
 
+    def test_getbuffer_gc_collect(self):
+        memio = self.ioclass(b"1234567890")
+        buf = memio.getbuffer()
+        memiowr = weakref.ref(memio)
+        bufwr = weakref.ref(buf)
+        # Create a reference loop.
+        a = [buf]
+        a.append(a)
+        # XXX The following code is not obvious because the current behaviour
```

Comment:
```suggestion
        # NB: The following code is not obvious because the current behaviour
```

Reply:
There are 460 XXX comments and only 8 NB comments. The numbers speak for themselves.

If this is a new generally accepted trend, and the XXX comments are considered harmful, it should be added in PEP 8 and PEP 7. But so far it doesn't look like it.

Reply:
```suggestion
        # The following code is not obvious because the current behaviour
```